index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Fred's Personal Portfolio</title>
+</head>
+<body>
+    <div class="header-container">
+        <header>
+            <nav class="nav-menu">
+                <a href="" class="logo"><img src="images/My-Logo.svg" alt=""></a>
+                <a href="" class="menu-icon"><img src="images/Icon-Menu.svg" alt=""></a>
+            </nav>
+            <div class="main-heading">
+                <h1>I’m Mariane Glad to see you!</h1>
+            </div>
+            <div>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+            </div>
+            <div>
+                <h3>LET'S CONNECT</h3>
+            </div>
+            <div class="social-icons">
+                <ul>
+                    <li><a href="#"><img src="images/Vector.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/Linkedin-icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/Vector1.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/Vector2.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/Vector3.svg" alt=""></a></li>
+                  </ul>
+            </div>
+            <div>
+                <img src="images/Rectangle5.png" class="rectangle" alt=""> 
+            </div>
+        </header>
```

Comment:
- [ ] Kindly reorganize your skeleton to be semantically accurate and fit the project requirements:

You can follow this adaptation of your current code to succeed on your next review.
```
        <header>
            <a href="" class="logo"><img src="images/My-Logo.svg" alt=""></a>
            <nav class="nav-menu">
                <a href="" class="menu-icon"><img src="images/Icon-Menu.svg" alt=""></a>
            </nav>
            <div>
                <img src="images/Rectangle5.png" class="rectangle" alt=""> 
            </div>
        </header> 
        <section>
              <div class="main-heading">
                <h1>I’m Mariane Glad to see you!</h1>
            </div>
            <div>
                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>

            </div>
            <div>
                <h3>LET'S CONNECT</h3>
            </div>
            <div class="social-icons">
                <ul>
                    <li><a href="#"><img src="images/Vector.svg" alt=""></a></li>
                    <li><a href="#"><img src="images/Linkedin-icon.svg" alt=""></a></li>
                    <li><a href="#"><img src="images/Vector1.svg" alt=""></a></li>
                    <li><a href="#"><img src="images/Vector2.svg" alt=""></a></li>
                    <li><a href="#"><img src="images/Vector3.svg" alt=""></a></li>
                  </ul>
            </div>
            <div>
                <img src="images/Rectangle5.png" class="rectangle" alt=""> 
            </div>
         </section>
```